src/main/java/uk/gov/companieshouse/company/profile/service/CompanyProfileService.java

```diff
@@ -129,4 +133,29 @@ public void updateInsolvencyLink(String contextId, String companyNumber,
         }
 
     }
+
```

Comment:
Updating only the fields mentioned in the ticket:
<img width="734" alt="image" src="https://user-images.githubusercontent.com/16381559/172841473-51bb1fd8-40ef-47ff-9ee8-2a6355449f2c.png">